client/src/components/MyAvailability.js

```diff
@@ -31,43 +33,29 @@ const MyAvailability = () => {
       <div>
         <NavBar />
       </div>
-      <h1
-        style={{
-          fontFamily: "sen",
-          color: "black",
-          textAlign: "center",
-          marginTop: "100px",
-          marginBottom: "50px",
-        }}
-      >
-        My Availability
-      </h1>
+      <h1>My Availability</h1>
```

Comment:
This could use the heading component that exists.

Reply:
Thank you. It's now using the heading component.